### DISCUSSION TITLE
Entità non disponibili

### DISCUSSION THREAD

**Discussion by dwarfish** [indomus/forum#3251] [HUB: Home Assistant]
Ciao a tutti,
ieri mi sono accorto che alcune entità risultano temporaneamente non disponibili per poi tornare disponibili senza che faccia nulla. Prendendo ad esempio l'entità Home Assistant Core Update da stamattina è successo già diverse volte:
![HA](https://github.com/user-attachments/assets/360456eb-a09d-4a8f-935f-109e6d6fd9cc)
Da che può dipendere?

Grazie,
^SiMoNe^

**Comment by KC-inDomus** [comment]
ci dici come hai risolto?

**Comment by dwarfish** [comment]
Mi dispiace, ma non ho risolto. Ho rifatto l'installazione da zero. Credo che il problema dipendesse dall'installazione iniziale sulla microSD e poi migrata sull'SSD esterno dalla GUI di Home Assistant (Impostazioni -> Archiviazione -> Sposta il disco dati) in quanto il RaspBerryPi andava spesso in blocco e il comando top restituiva valori anomali di attesa (wa 90%).

^SiMoNe^
